src/app/suiteDb/custom-materials/custom-materials.component.html

```diff
@@ -29,14 +29,15 @@
     <div class="tab-content content-container">
       <div class="card-form">
         <label class="group-label general" (click)="toggleMaterial('flue')">Custom Gaseous Fuels
-          <span class="fa fa-chevron-up pull-right" *ngIf="!showFlueMaterials"></span>
-          <span class="fa fa-chevron-down pull-right" *ngIf="showFlueMaterials"></span>
+          <span class="fa fa-chevron-down pull-right m-1" *ngIf="!showFlueMaterials"></span>
+          <span class="fa fa-chevron-up pull-right m-1" *ngIf="showFlueMaterials && hasFlueMaterials"></span>
         </label>
         <a class="click-link pull-right small" *ngIf="showFlueMaterials" (click)="toggleModal('flue')">+Add Custom Gaseous Fuel</a>
         <!-- <p class="help-text">View, create, edit your custom flue gas materials.</p> -->
       </div>
     </div>
-    <app-custom-flue-gas-materials *ngIf="showFlueMaterials" [settings]="settings" [showModal]="showFlueModal" [importing]="importing"></app-custom-flue-gas-materials>
+    <app-custom-flue-gas-materials *ngIf="showFlueMaterials" [settings]="settings" [showModal]="showFlueModal" [importing]="importing"
+      (emitNumMaterials)="hasFlueMaterials = ($event > 0)"></app-custom-flue-gas-materials>
```

Comment:
We don't want to do logic for setting variables in the html template. Create a function that is called in the .ts

Reply:
Fixed